spring-petclinic-api-gateway/src/main/resources/application.yml

```diff
@@ -5,6 +5,16 @@ spring:
     import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
   cloud:
     gateway:
+      default-filters:
+        - name: CircuitBreaker
+          args:
+            name: defaultCircuitBreaker
+            fallbackUri: forward:/fallback
+        - name: Retry
+          args:
+            retries: 1
+            statuses: SERVICE_UNAVAILABLE
+            methods: GET, POST
```

Comment:
The `POST` looks like to be enough. Isn't it?

Reply:
Seems like it, yes.